ubuntu-advantage.1

```diff
@@ -170,6 +170,16 @@ downtime, keeping your Ubuntu LTS systems secure and compliant.
 
 See https://ubuntu.com/livepatch for more information.
 
+.SH EXIT STATUS
+The Ubuntu Advantage command line tool will exit with the following codes:
+
+.BR 0: " Success"
+.br
+.BR 1: " Error occurred during the operation"
+Error messages are emitted to console and /var/log/ubuntu-advantage.log
+.br
+.BR 2: " Non-critical warning occurred during the operation"
```

Comment:
Warnings are emitted, they don't occur. Perhaps s/warning/error/, and s/Error/Critical error/ in the `1` description?

Reply:
good thought. done